zkevm-circuits/src/evm_circuit/execution/error_oog_call.rs

```diff
@@ -65,15 +60,16 @@ impl<F: Field> ExecutionGadget<F> for ErrorOOGCallGadget<F> {
             is_warm.expr(),
         );
 
-        let balance = cb.query_word_rlc();
-        cb.account_read(
-            call_gadget.callee_address_expr(),
-            AccountFieldTag::Balance,
-            balance.expr(),
-        );
+        // TODO: Why is calle balance read here? There's no constraint related to this
```

Comment:
We can safely remove the balance read here. The old OOG callop uses zero nonce, zero balance, and empty code hash to determine the empty account. In #1023, balance and nonce check are removed, but the balance read was a leftover. 

Reference: https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1023#discussion_r1066505492

Reply:
ah thanks for the explanation! I've removed the commented code :)